index.js

```diff
@@ -48,32 +44,20 @@ app.post('/', function (req, res) {
 
     // ensure content type is set
     if (!req.headers['content-type']) {
-        res.status(500).send('Content-Type must be set, use text/plain if unsure')
+        res.status(500).send('Content-Type must be set, use application/json if unsure')
```

Comment:
```suggestion
        res.status(400).send('Content-Type must be set, use application/json if unsure')
```

technically not a server error